ps_mbo.php

```diff
@@ -234,18 +231,21 @@ public function fetchModulesByController($ajax = false)
     }
     
     protected function _handleAddonsConnectWithMbo() {
-        $return = false;
+        if (Tools::getIsset('controller') && Tools::getValue('controller') != 'AdminPsMboModule') {
+            return false;
+        }
+        
         if (Tools::getIsset('controller') && Tools::getValue('controller') == 'AdminPsMboModule') {
```

Comment:
You add a condition to return false as fast as possible, this one is now useless. Same for the last return false;

Reply:
i removed the first if, i need the second one